bob/build_nix.go

```diff
@@ -3,14 +3,30 @@ package bob
 import (
 	"fmt"
 
+	"github.com/benchkram/errz"
+
 	"github.com/benchkram/bob/bob/bobfile"
+	cache2 "github.com/benchkram/bob/pkg/cache"
```

Comment:
Why cache2?

Reply:
I think during refactor I had 2 caches there and forgot to remove. Will remove